RDSExpoApp/app/(tabs)/_layout.tsx

```
@@ -0,0 +1,35 @@
+          title: 'index',
+          tabBarIcon: ({ color }) => (
+            <FontAwesome name="code" size={28} color={color} style={{ marginBottom: -3 }} />
+          ),
+        }}
+      />
+      <Tabs.Screen
+        name="ProfileScreen"
+        options={{
+          title: 'ProfileScreen',
+          tabBarIcon: ({ color }) => (
+            <FontAwesome name="code" size={28} color={color} style={{ marginBottom: -3 }} />
```

ESLint

[error] Replace `·name="code"·size={28}·color={color}·style={{·marginBottom:·-3·}}` with `⏎··············name="code"⏎··············size={28}⏎··············color={color}⏎··············style={{·marginBottom:·-3·}}⏎···········`

(prettier/prettier)